src/components/Header/HeaderUserMenu/HeaderUserMenu.js

```diff
@@ -0,0 +1,16 @@
+import React, { Component } from 'react';
+
+class HeaderUserMenu extends Component {
```

Comment:
What do you think about simply using a function in place of a component?

Reply:
it's depends if need or not state. I not implemented yet this 💃